src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -155,7 +178,61 @@
 				<PackageReference Include="Uno.Extensions.Logging.OSLog" Version="1.4.0" />
 			</ItemGroup>
 			<ItemGroup>
-				<LinkDescription Include="iOS\LinkerExclusions.xml" />
+				<LinkDescription Include="iOS/LinkerExclusions.xml" />
+			</ItemGroup>
+			<ItemGroup>
+				<BundleResource Include="iOS/Resources/Assets/splashscreen_icon.png" />
+				<BundleResource Include="iOS/Resources/Assets/splashscreen_icon%402x.png" />
+				<BundleResource Include="iOS/Resources/Assets/splashscreen_icon%403x.png" />
+				<InterfaceDefinition Include="iOS/LaunchScreen.storyboard" />
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Contents.json">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-1024.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-20.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-20@2x.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-20@3x.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-29.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-29@2x.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-29@3x.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-40.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-40@2x.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-40@3x.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-60@2x.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-60@3x.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-76.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\AppIcons.appiconset\Icon-76@2x.png">
+					<Visible>false</Visible>
+				</ImageAsset>
+				<ImageAsset Include="iOS\Media.xcassets\LaunchImages.launchimage\Contents.json">
+					<Visible>false</Visible>
+				</ImageAsset>
 			</ItemGroup>
```

Comment:
I don't even know if this was necessary, it was working fine without it locally.